src/test/java/com/tourguide/myclassjournal/TourGuideApplicationTests.java

```diff
@@ -0,0 +1,14 @@
+package com.tourguide.myclassjournal;
+
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class TourGuideApplicationTests {
+    
+    @Test
+    void contextLoads(){
+
+    }
+}
```

Comment:
There were not any tests when I went to look at the project, nor was there test files stemming from the src directory. So I had manually created TourGuideApplicationTests as well as making it stem from the src directory in an organized manner.
Source: [https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)

Reply:
The test is here, but empty.  Avoid committing unimplemented code to main, even if it is just a test.